.gitignore

```diff
@@ -446,3 +446,4 @@ Brewfile.lock.json
 # GPerf heap profile dumps
 *.heap
 *.mo
+/android-project/app/src/main/assets/*.gmo
```

Comment:
The rule to ignore the TTF font file is in `android-project/.gitignore`.
https://github.com/diasurgical/devilutionX/blob/cfa3c8fa4e08e19a1976fd64cbc1472762e9d150/android-project/.gitignore#L14